scikeras/wrappers.py

```diff
@@ -1257,7 +1279,7 @@ def __init__(
         ] = "rmsprop",
         loss: Union[
             Union[str, tf.keras.losses.Loss, Type[tf.keras.losses.Loss], Callable], None
-        ] = None,
+        ] = "auto",
```

Comment:
TODO: bring in the nice type-hint changes from #208 into a future PR (no need to put it in here)

Reply:
:+1: I added the types because it made `KerasRegressor.__init__` easier to implement.

I think `test_simple_usage.py` should also be brought in too.

Reply:
`test_simple_usage.py` is brought in.

I do like the types, but I think it can be done in a future PR to keep the intent here clear (even if it results in a larger diff because`__init__` is ugly).

Reply:
Edit: I did not push `test_simple_usage.py`. I'll push it now. Sorry for the confusion.

Reply:
Pushed.

Reply:
Tracking type-hints in https://github.com/adriangb/scikeras/issues/211